packages/SettingsProvider/src/com/android/providers/settings/SettingsProvider.java

```diff
@@ -4882,6 +4882,16 @@ private int onUpgradeLocked(int userId, int oldVersion, int newVersion) {
                     currentVersion = 191;
                 }
 
+                if (currentVersion == 191) {
+                        // Version 191: Update auto reboot constant string for GrapheneOS
+                        public static final String OLD_TIMEOUT_STRING = "settings_reboot_after_timeout";
+                        final long timeoutVal = Settings.Global.getLong(mContext.getContentResolver(), OLD_TIMEOUT_STRING, 0);
+                        if (timeoutVal != 0) {
+                                Settings.Global.putLong(mContext.getContentResolver(), Settings.Global.SETTINGS_REBOOT_AFTER_TIMEOUT, timeoutVal);
+                        }
+                        currentVersion = 192;
```

Comment:
I don't really want to change this ourselves. It's going to conflict with upstream changes.

Reply:
fixed